src/fetch/transfer_file.py

```diff
@@ -389,35 +398,69 @@ def unusual_slippage_url(period: AccountingPeriod) -> str:
     return base + urllib.parse.quote_plus(query, safe="=&?")
 
 
+def summary(transfers: list[Transfer]) -> str:
+    """Summarizes transfers with totals"""
+    eth_total = sum(t.amount_wei for t in transfers if t.token_type == TokenType.NATIVE)
+    cow_total = sum(t.amount_wei for t in transfers if t.token_type == TokenType.ERC20)
+    return (
+        f"Total ETH Funds needed: {eth_total / 10 ** 18}\n"
+        f"Total COW Funds needed: {cow_total / 10 ** 18}\n"
+    )
+
+
 def manual_propose(dune: DuneAPI, period: AccountingPeriod) -> None:
     """
     Entry point to manual creation of rewards payout transaction.
     This function generates the CSV transfer file to be pasted into the COW Safe app
     """
     print(
-        f"While you are waiting, The data being compiled here can be visualized at\n"
-        f"{dashboard_url(period)}\n"
-        f"In particular, please double check the batches with unusual slippage: "
+        f"Please double check the batches with unusual slippage: "
         f"{unusual_slippage_url(period)}"
     )
     transfers = consolidate_transfers(get_transfers(dune, period))
     write_to_csv(
         data_list=[CSVTransfer.from_transfer(t) for t in transfers],
         outfile=File(name=f"transfers-{period}.csv"),
     )
-    eth_total = sum(t.amount_wei for t in transfers if t.token_type == TokenType.NATIVE)
-    cow_total = sum(t.amount_wei for t in transfers if t.token_type == TokenType.ERC20)
-
+    print(summary(transfers))
     print(
-        f"Total ETH Funds needed: {eth_total / 10 ** 18}\n"
-        f"Total COW Funds needed: {cow_total / 10 ** 18}\n"
         f"Please cross check these results with the dashboard linked above.\n "
         f"For solver payouts, paste the transfer file CSV Airdrop at:\n"
-        f"{safe_url()}"
+        f"{AIRDROP_URL}"
     )
 
 
-def auto_propose(dune: DuneAPI, period: AccountingPeriod) -> None:
+def post_to_slack(
+    slack_client: WebClient, channel: str, message: str, sub_message: str
+) -> None:
+    """Posts message to slack channel and sub message inside thread of first message"""
+    response = slack_client.chat_postMessage(
+        channel=channel,
+        text=message,
+        # Do not show link preview!
+        # https://api.slack.com/reference/messaging/link-unfurling
+        unfurl_media=False,
+    )
+    # This assertion is only for type safety,
+    # since previous function can also return a Future
+    assert isinstance(response, SlackResponse)
+    # Post logs in thread.
+    slack_client.chat_postMessage(
+        channel=channel,
+        format="mrkdwn",
+        text=sub_message,
+        # According to https://api.slack.com/methods/conversations.replies
+        thread_ts=response.get("ts"),
+        unfurl_media=False,
+    )
+    # return response.get("ts"), sub_response.get("ts")
```

Comment:
Forgotten comment?

Reply:
Kinda. I am actually still considering Writing a cleanup option in our unit tests but do do this, the post_to_slack would have to return a message ID that can be deleted after. Unfortunately these values are not deterministic, so I would also have to come up with a better way to test that the message was sent (instead of assert = None <-- the current return value of the method). I have been playing with the idea of returning the IDs of the successfully posted messages so that we can refer back to them. Still not sure yet what to do about this. I might actually leave this comment and add a TODO with some more context here for a follow up PR.

Reply:
Created an issue for this: https://github.com/cowprotocol/solver-rewards/issues/81